packages/esm-patient-orders-app/translations/fr.json

```diff
@@ -3,7 +3,7 @@
   "activeVisitRequired": "Une visite ouverte est nécessaire pour prescrire",
   "add": "Ajouter",
   "additionalInstructions": "Instructions supplémentaires",
-  "addOrderableForOrderType": "Ajouter {{orderTypeDisplay}}",
+  "addOrderableForOrderType": "{{orderTypeDisplay}}",
```

Comment:
@rbuisson we should maintain `Ajouter` in this string, right?

Reply:
Hi @denniskigen , actually no.
Since the orderTypeDisplay value in French can be either grammatically feminine or masculine (eg, f: procedure, m: scan), then we would need to have "Ajouter un" or "Ajouter une". ("Ajouter" alone does not work).
Alternatively, we could do "Ajouter un(e)".

But I think overall, removing "Ajouter" is probably fine in this context, since it's not an action button, but a title of the workspace. Slight divergence from the English version, but I hope that's fine.

Reply:
Ah, I see! Thanks for the context, @rbuisson!